baseplate/server/__init__.py

```diff
@@ -75,7 +75,7 @@ def configure_logging(debug):
 
 def make_listener(endpoint):
     try:
-        fileno = int(os.environ["EINHORN_FDS"])
+        fd_count = int(os.environ["EINHORN_FD_COUNT"])
     except KeyError:
```

Comment:
What's the catch on the `KeyError` for here?  To detect whether or not we're running under einhorn?